modules/update-service-overview.adoc

```diff
@@ -11,34 +11,34 @@
 
 [IMPORTANT]
 ====
-Because the OpenShift Update Service displays all valid updates, you must not force an update to a version that the OpenShift Update Service does not display.
+The OpenShift Update Service displays all valid updates. Do not force an update to a version that the OpenShift Update Service does not display.
 ====
 
 ////
 The interaction between the registry and the OpenShift Update Service is different during bootstrap and continuous update modes. When you bootstrap the initial infrastructure, the Cluster Version Operator finds the fully qualified image name for the shortname of the images that it needs to apply to the server during installation. It looks at the imagestream that it needs to apply and renders it to disk. It calls bootkube and waits for a temporary minimal control plane to come up and load the Cluster Version Operator.
 ////
 
-During continuous update mode, two controllers run. One continuously updates the payload manifests, applies them to the cluster, and outputs the status of the controlled rollout of the Operators, whether they are available, upgrading, or failed. The second controller polls the OpenShift Update Service to determine if updates are available.
+Two controllers run during continuous update mode. The first controller continuously updates the payload manifests, applies the manifests to the cluster, and outputs the controlled rollout status of the Operators to indicate whether they are available, upgrading, or failed. The second controller polls the OpenShift Update Service to determine if updates are available.
 
 [IMPORTANT]
 ====
-Reverting your cluster to a previous version, or a rollback, is not supported.
-Only upgrading to a newer version is supported.
-If your upgrade fails, contact Red Hat support.
+Only upgrading to a newer version is supported. Reverting or rolling back your cluster to a previous version is not supported. If your upgrade fails, contact Red Hat support.
 ====
 
-During the upgrade process, the Machine Config Operator (MCO) applies the new configuration to your cluster machines. It cordons the number of nodes that is specified by the `maxUnavailable` field on the machine configuration pool and marks them as unavailable. By default, this value is set to `1`. It then applies the new configuration and reboots the machine. If you use Red Hat Enterprise Linux (RHEL) machines as workers, the MCO does not update the kubelet on these machines because you must update the OpenShift API on them first. Because the specification for the new version is applied to the old kubelet, the RHEL machine cannot return to the `Ready` state. You cannot complete the update until the machines are available. However, the maximum number of nodes that are unavailable is set to ensure that normal cluster operations are likely to continue with that number of machines out of service.
```

Comment:
You dropped this key distinguishing condition: `If you use Red Hat Enterprise Linux (RHEL) machines as workers`

Reply:
Fair point. I see where my change in wording might have made this less clear: `The MCO does not update the kubelet on Red Hat Enterprise Linux (RHEL) machines if you use them as workers...`

I've changed it back to:
>If you use Red Hat Enterprise Linux (RHEL) machines as workers, the MCO does not update the kubelet because you must update the OpenShift API on the machines first.